csci_tool/grader.py

```diff
@@ -30,10 +30,14 @@ class Grader():
                 student (Student)
                 source_dir (PathLike): directory where grade.py lives
             Returns:
-                tuple (int, int): (score, max_score)
+                tuple (list, int): (list_of_scores, max_score)
```

Comment:
what is the list of scores?

Reply:
So I thought it would be better to be able to return multiple scores. So for the first project the grader returns the Correct Score, + the performance score. Then each of them are printed into the csv. This way we can see a more detailed breakdown of the scores.

Reply:
Ah got it that makes sense. Maybe we should add a function that gets labels so that we can write that into a header in the csv file